.ci/run_rhub_solaris_checks.R

```diff
@@ -9,22 +9,30 @@ email <- c(
     150L, 147L, 145L, 146L, 158L, 145L, 140L, 151L, 137L, 156L, 146L, 159L, 140L, 137L, 141L, 146L,
     143L, 141L, 149L, 157L, 106L, 163L, 153L, 154L, 151L, 139L, 147L, 150L, 88L, 141L, 153L, 151L
 )
-rhub::validate_email(
-    email = intToUtf8(email - 42L)
-    , token = "181dd96184924f0aaea19d73f9ae7974"
+token = c(
+    91L, 98L, 91L, 142L, 142L, 99L, 96L, 91L, 98L, 94L, 99L, 92L, 94L, 144L, 90L, 139L,
+    139L, 143L, 139L, 91L, 99L, 142L, 97L, 93L, 144L, 99L, 139L, 143L, 97L, 99L, 97L, 94L
 )
 
 if (Sys.info()["sysname"] == "Windows") {
     null_file <- "NUL"
 } else {
     null_file <- "/dev/null"
 }
+
 sink(file = null_file)
+rhub::validate_email(
+    email = intToUtf8(email - 42L)
+    , token = intToUtf8(token - 42L)
+)
+sink()
```

Comment:
Prevent e-mail is being printed in logs. I believe that bots can also parse logs for e-mails printed in pain text.

![image](https://user-images.githubusercontent.com/25141164/107148817-5c3cc800-6966-11eb-89be-b29f8314859b.png)

Reply:
ooo good catch, didn't think about that